### ISSUE TITLE
MUST be specified check too strict

### ISSUE THREAD

**Issue by thijskh** [baszoetekouw/janus-py#4]
The code:

```python
              for field in ('allowedConnection','blockedConnections','disableConsentConnections','arpAttributes','manipulationCode'):
                        if not field in newentity:
                                raise ServiceRegistryError(-1,"%s MUST be specified, otherwise the API will be silently truncate it" % field)
```

is too strict because it's perfectly normal for an entity to have no AM. Also, it always triggers since 'allowedConnection' misses an 's'.

**Comment by baszoetekouw** [comment]
I put the check in there, because without it, you might accidentally overwrite an existing AM or connection list.  Replacing the method name by replaceEntityById() might solve the issue.

**Comment by thijskh** [comment]
Right, but the update() method arranges this for you properly. Most entities will not have any manipulationcode so this will fail.

I use update like this:
```
newfield['metadata']['example'] = 25
sr.update(eid, newfield,note="Zet op 25")
```
This always fails for entities without current manipulationCode and there's no way to override it, while this option does seem safe to execute.
